src/meltano/cli/elt.py

```diff
@@ -103,6 +108,29 @@ async def elt(
 
     \b\nRead more at https://docs.meltano.com/reference/command-line-interface#elt
     """
+    tracker = Tracker(project)
+    cmd_ctx = cli_context_builder(
+        "elt",
+        None,
+        dry=dry,
+        transform=transform,
+        full_refresh=full_refresh,
+        select=select,
+        exclude=exclude,
+        catalog=catalog,
+        state=state,
+        dump=dump,
+        state_id=state_id,
+        force=force,
+    )
+    tracker.add_contexts(cmd_ctx)
+    tracker.track_command_event(cli_tracking.STARTED)
+
+    # we no longer set a default choice for transform, so that we can detect explicit usages of the --transform option
+    # if transform is None we still need manually default to skip after firing the tracking event above.
+    if not transform:
+        transform = "skip"
```

Comment:
@aaronsteers @pnadolny13 this should do it. Since `transform` was click.Choice it never actually printed what the default choice was in the help string as far as I could tell. 

So I just omit the default in the click.option definition and then override it here. But above we'll correctly only include the transform option in the tracking event if it was actually set by the user.